src/bitcoind-res.rc

```diff
@@ -17,13 +17,13 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoind (OSS daemon/client for Bitcoin)"
+            VALUE "FileDescription",    "bitcoind (Bitcoin client as a JSON-RPC server)"
```

Comment:
It's the only user interface... "headless" or "without GUI" implies it is suitable for human use.